src/transactions/test/ChangeTrustTests.cpp

```diff
@@ -799,9 +799,6 @@ TEST_CASE("change trust pool share trustline",
                             .baseReserve == newReserve);
                 };
 
-                auto usd = makeAsset(gateway, "USD");
-                auto idrUsd = makeChangeTrustAssetPoolShare(
```

Comment:
Have you checked that this is supposed to be dead code? r=me if so but it seems like it's worth confirming